ibm/mas_devops/roles/suite_install/defaults/main.yml

```diff
@@ -9,6 +9,9 @@ mas_channel: "{{ lookup('env', 'MAS_CHANNEL') }}"
 mas_domain: "{{ lookup('env', 'MAS_DOMAIN') }}"
 mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
 
+# Special characters
```

Comment:
This needs to cater for older versions of MAS where this settings does not exist.

e.g. should default to undefined/""

```
mas_special_characters : "{{ lookup('env', 'MAS_SPECIAL_CHARACTERS') }}"
```

User must set it to true|false, if they don't it's "", which means we do nothing.

Reply:
"This should be sufficient, right? If we're validating the condition in the [ibm/mas_devops/roles/suite_install/templates/core_v1_suite.yml.j2](https://github.com/ibm-mas/ansible-devops/pull/1412/files#diff-43635fa581ff608d63be293b3806a4f3045dfaad79a9c48e1a739b90f177cc9a) file, it should be enough to prevent the issue you mentioned.